pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -247,11 +259,12 @@ let
         "--config=mkl_open_source_only"
       ];
 
-      sha256 =
-        if cudaSupport then
-          "sha256-O6bM7Lc8eaFyO4Xzl5/hvBrbPioI+Yeqx9yNC97fvKk="
-        else
-          "sha256-gLMJfJSQIdGGY2Ivx4IgDWg0hc+mxzlqY11CUkSWcjI=";
+      sha256 = (if cudaSupport then {
+        x86_64-linux = "sha256-8QaXoZq6oITRsYn4RdLUXcKQv3PJ4Q3ItX9PkBwxGBI=";
+      } else {
+        x86_64-linux = "sha256-M/h5EZmyiV4QvzgKRjdz7V1LHENUJlc/ig1QAItnWVQ=";
+        aarch64-linux = "sha256-edkYcdlvOLNGRSanch1fGCZwq8SFn3TzcUNt1LhzG/E=";
+      }).${stdenv.system} or (throw "jaxlib: unsupported system: ${stdenv.system}");
```

Comment:
@uri-canva previously went through the trouble to avoid this kerfuffle in https://github.com/NixOS/nixpkgs/pull/221390. why does that break now? can we adjust the fetchAttrs such that the hashes match?

Reply:
Maybe be I'm not really interested in going down that rabbit hole... The whole bazel fetching thing is already a nightmare.